health/notifications/alarm-notify.sh.in

```diff
@@ -3485,7 +3498,7 @@ Content-Transfer-Encoding: 8bit
                     <tbody>
                     <tr>
                       <td align="left" style="font-size:0px;padding:10px 25px;padding-top:0;padding-bottom:0;word-break:break-word;">
-                        <div style="font-family:Open Sans, sans-serif;font-size:13px;line-height:1;text-align:center;color:#35414A;">© Netdata 2021 - The real-time performance and health monitoring</div>
+                        <div style="font-family:Open Sans, sans-serif;font-size:13px;line-height:1;text-align:center;color:#35414A;">© Netdata 2022 - The real-time performance and health monitoring</div>
```

Comment:
You are updating the year, so it looks like for me that we should update this every year. If this conclusion is correct, instead `2022`, please, use `date +'%Y'`.

Reply:
Yes, have it in mind, cool trick I will check for auto date there. Thanks!

Reply:
Done!